File: samples/client/3_0_3_unit_test/java/src/main/java/org/openapijsonschematools/client/schemas/validation/JsonSchemaInfo.java
package org.openapijsonschematools.client.schemas.validation;
import org.checkerframework.checker.nullness.qual.NonNull;
import org.checkerframework.checker.nullness.qual.Nullable;

import java.math.BigDecimal;
import java.util.List;
import java.util.Map;
import java.util.Set;
import java.util.regex.Pattern;

public class JsonSchemaInfo {
    public @Nullable Set<Class<?>> type = null;
    public JsonSchemaInfo type(@NonNull Set<Class<?>> type) {
        this.type = type;
        return this;
    }
    public @Nullable String format = null;
    public JsonSchemaInfo format(@NonNull String format) {
        this.format = format;
        return this;
    }
    public @Nullable Class<? extends JsonSchema> items = null;
    public JsonSchemaInfo items(@NonNull Class<? extends JsonSchema> items) {
        this.items = items;
        return this;
    }
    public @Nullable Map<String, Class<? extends JsonSchema>> properties = null;
    public JsonSchemaInfo properties(@NonNull Map<String, Class<? extends JsonSchema>> properties) {
        this.properties = properties;
        return this;
    }
    public @Nullable Set<String> required = null;
    public JsonSchemaInfo required(@NonNull Set<String> required) {
        this.required = required;
        return this;
    }
    public @Nullable Number exclusiveMaximum = null;
    public JsonSchemaInfo exclusiveMaximum(@NonNull Number exclusiveMaximum) {
        this.exclusiveMaximum = exclusiveMaximum;
        return this;
    }
    public @Nullable Number exclusiveMinimum = null;
    public JsonSchemaInfo exclusiveMinimum(@NonNull Number exclusiveMinimum) {
        this.exclusiveMinimum = exclusiveMinimum;
        return this;
    }
    public @Nullable Integer maxItems = null;
    public JsonSchemaInfo maxItems(@NonNull Integer maxItems) {
        this.maxItems = maxItems;
        return this;
    }
    public @Nullable Integer minItems = null;
    public JsonSchemaInfo minItems(@NonNull Integer minItems) {
        this.minItems = minItems;
        return this;
    }
    public @Nullable Integer maxLength = null;
    public JsonSchemaInfo maxLength(@NonNull Integer maxLength) {
        this.maxLength = maxLength;
        return this;
    }
    public @Nullable Integer minLength = null;
    public JsonSchemaInfo minLength(@NonNull Integer minLength) {
        this.minLength = minLength;
        return this;
    }
    public @Nullable Integer maxProperties = null;
    public JsonSchemaInfo maxProperties(@NonNull Integer maxProperties) {
        this.maxProperties = maxProperties;
        return this;
    }
    public @Nullable Integer minProperties = null;
    public JsonSchemaInfo minProperties(@NonNull Integer minProperties) {
        this.minProperties = minProperties;
        return this;
    }
    public @Nullable Number maximum = null;
    public JsonSchemaInfo maximum(@NonNull Number maximum) {
        this.maximum = maximum;
        return this;
    }
    public @Nullable Number minimum = null;
    public JsonSchemaInfo minimum(@NonNull Number minimum) {
        this.minimum = minimum;
        return this;
    }
    public @Nullable BigDecimal multipleOf = null;
    public JsonSchemaInfo multipleOf(@NonNull BigDecimal multipleOf) {
        this.multipleOf = multipleOf;
        return this;
    }
    public @Nullable Class<? extends JsonSchema> additionalProperties;
    public JsonSchemaInfo additionalProperties(@NonNull Class<? extends JsonSchema> additionalProperties) {
        this.additionalProperties = additionalProperties;
        return this;
    }
    public @Nullable List<Class<? extends JsonSchema>> allOf = null;
    public JsonSchemaInfo allOf(@NonNull List<Class<? extends JsonSchema>> allOf) {
        this.allOf = allOf;
        return this;
    }
    public @Nullable List<Class<? extends JsonSchema>> anyOf = null;
    public JsonSchemaInfo anyOf(@NonNull List<Class<? extends JsonSchema>> anyOf) {
        this.anyOf = anyOf;
        return this;
    }
    public @Nullable List<Class<? extends JsonSchema>> oneOf = null;
    public JsonSchemaInfo oneOf(@NonNull List<Class<? extends JsonSchema>> oneOf) {
        this.oneOf = oneOf;
        return this;
    }
    public @Nullable Class<? extends JsonSchema> not = null;
    public JsonSchemaInfo not(@NonNull Class<? extends JsonSchema> not) {
        this.not = not;
        return this;
    }
    public @Nullable Boolean uniqueItems = null;
    public JsonSchemaInfo uniqueItems(@NonNull Boolean uniqueItems) {
        this.uniqueItems = uniqueItems;
        return this;
    }
    public @Nullable Set<Object> enumValues = null;
    public JsonSchemaInfo enumValues(@NonNull Set<Object> enumValues) {
        this.enumValues = enumValues;
        return this;
    }
    public @Nullable Pattern pattern = null;
    public JsonSchemaInfo pattern(@NonNull Pattern pattern) {
        this.pattern = pattern;
        return this;
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/schemas/validation/JsonSchemaInfo.kt
package org.openapijsonschematools.client.schemas.validation

import java.math.BigDecimal
import java.util.regex.Pattern

class JsonSchemaInfo {
    var type: Set<Class<*>>? = null
    fun type(type: Set<Class<*>>?): JsonSchemaInfo {
        this.type = type
        return this
    }

    var format: String? = null
    fun format(format: String?): JsonSchemaInfo {
        this.format = format
        return this
    }

    var items: Class<out JsonSchema<*>?>? = null
    fun items(items: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.items = items
        return this
    }

    var properties: Map<String, Class<out JsonSchema<*>?>>? = null
    fun properties(properties: Map<String, Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.properties = properties
        return this
    }

    var required: Set<String>? = null
    fun required(required: Set<String>?): JsonSchemaInfo {
        this.required = required
        return this
    }

    var exclusiveMaximum: Number? = null
    fun exclusiveMaximum(exclusiveMaximum: Number?): JsonSchemaInfo {
        this.exclusiveMaximum = exclusiveMaximum
        return this
    }

    var exclusiveMinimum: Number? = null
    fun exclusiveMinimum(exclusiveMinimum: Number?): JsonSchemaInfo {
        this.exclusiveMinimum = exclusiveMinimum
        return this
    }

    var maxItems: Int? = null
    fun maxItems(maxItems: Int?): JsonSchemaInfo {
        this.maxItems = maxItems
        return this
    }

    var minItems: Int? = null
    fun minItems(minItems: Int?): JsonSchemaInfo {
        this.minItems = minItems
        return this
    }

    var maxLength: Int? = null
    fun maxLength(maxLength: Int?): JsonSchemaInfo {
        this.maxLength = maxLength
        return this
    }

    var minLength: Int? = null
    fun minLength(minLength: Int?): JsonSchemaInfo {
        this.minLength = minLength
        return this
    }

    var maxProperties: Int? = null
    fun maxProperties(maxProperties: Int?): JsonSchemaInfo {
        this.maxProperties = maxProperties
        return this
    }

    var minProperties: Int? = null
    fun minProperties(minProperties: Int?): JsonSchemaInfo {
        this.minProperties = minProperties
        return this
    }

    var maximum: Number? = null
    fun maximum(maximum: Number?): JsonSchemaInfo {
        this.maximum = maximum
        return this
    }

    var minimum: Number? = null
    fun minimum(minimum: Number?): JsonSchemaInfo {
        this.minimum = minimum
        return this
    }

    var multipleOf: BigDecimal? = null
    fun multipleOf(multipleOf: BigDecimal?): JsonSchemaInfo {
        this.multipleOf = multipleOf
        return this
    }

    var additionalProperties: Class<out JsonSchema<*>?>? = null
    fun additionalProperties(additionalProperties: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.additionalProperties = additionalProperties
        return this
    }

    var allOf: List<Class<out JsonSchema<*>?>>? = null
    fun allOf(allOf: List<Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.allOf = allOf
        return this
    }

    var anyOf: List<Class<out JsonSchema<*>?>>? = null
    fun anyOf(anyOf: List<Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.anyOf = anyOf
        return this
    }

    var oneOf: List<Class<out JsonSchema<*>?>>? = null
    fun oneOf(oneOf: List<Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.oneOf = oneOf
        return this
    }

    var not: Class<out JsonSchema<*>?>? = null
    fun not(not: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.not = not
        return this
    }

    var uniqueItems: Boolean? = null
    fun uniqueItems(uniqueItems: Boolean?): JsonSchemaInfo {
        this.uniqueItems = uniqueItems
        return this
    }

    var enumValues: Set<Any>? = null
    fun enumValues(enumValues: Set<Any>?): JsonSchemaInfo {
        this.enumValues = enumValues
        return this
    }

    var pattern: Pattern? = null
    fun pattern(pattern: Pattern?): JsonSchemaInfo {
        this.pattern = pattern
        return this
    }

    var defaultValue: Any? = null
    var defaultValueSet = false
    fun defaultValue(defaultValue: Any?): JsonSchemaInfo {
        this.defaultValue = defaultValue
        defaultValueSet = true
        return this
    }

    var constValue: Any? = null
    var constValueSet = false
    fun constValue(constValue: Any?): JsonSchemaInfo {
        this.constValue = constValue
        constValueSet = true
        return this
    }

    var contains: Class<out JsonSchema<*>?>? = null
    fun contains(contains: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.contains = contains
        return this
    }

    var maxContains: Int? = null
    fun maxContains(maxContains: Int?): JsonSchemaInfo {
        this.maxContains = maxContains
        return this
    }

    var minContains: Int? = null
    fun minContains(minContains: Int?): JsonSchemaInfo {
        this.minContains = minContains
        return this
    }

    var propertyNames: Class<out JsonSchema<*>?>? = null
    fun propertyNames(propertyNames: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.propertyNames = propertyNames
        return this
    }

    var dependentRequired: Map<String, Set<String>>? = null
    fun dependentRequired(dependentRequired: Map<String, Set<String>>?): JsonSchemaInfo {
        this.dependentRequired = dependentRequired
        return this
    }

    var dependentSchemas: Map<String, Class<out JsonSchema<*>?>>? = null
    fun dependentSchemas(dependentSchemas: Map<String, Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.dependentSchemas = dependentSchemas
        return this
    }

    var patternProperties: Map<Pattern, Class<out JsonSchema<*>?>>? = null
    fun patternProperties(patternProperties: Map<Pattern, Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.patternProperties = patternProperties
        return this
    }

    var prefixItems: List<Class<out JsonSchema<*>?>>? = null
    fun prefixItems(prefixItems: List<Class<out JsonSchema<*>?>>?): JsonSchemaInfo {
        this.prefixItems = prefixItems
        return this
    }

    var ifSchema: Class<out JsonSchema<*>?>? = null
    fun ifSchema(ifSchema: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.ifSchema = ifSchema
        return this
    }

    var then: Class<out JsonSchema<*>?>? = null
    fun then(then: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.then = then
        return this
    }

    var elseSchema: Class<out JsonSchema<*>?>? = null
    fun elseSchema(elseSchema: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.elseSchema = elseSchema
        return this
    }

    var unevaluatedItems: Class<out JsonSchema<*>?>? = null
    fun unevaluatedItems(unevaluatedItems: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.unevaluatedItems = unevaluatedItems
        return this
    }

    var unevaluatedProperties: Class<out JsonSchema<*>?>? = null
    fun unevaluatedProperties(unevaluatedProperties: Class<out JsonSchema<*>?>?): JsonSchemaInfo {
        this.unevaluatedProperties = unevaluatedProperties
        return this
    }
}